Package.swift

```diff
@@ -101,25 +106,29 @@ func windowsLinkerSettings() -> [LinkerSetting] {
   let libs = pkgConfigValues(in: t, for: "Libs")
   let linkLibraries = libs.lazy.filter { $0.starts(with: "-l") || $0.first != "-" }.map {
     let rest = $0.dropFirst($0.first == "-" ? 2 : 0)
-    let afterSlashes = rest.lastIndex(where: { $0 == "/" || $0 == "\\" })
+    let afterSlashes =
+      rest.lastIndex(where: { $0 == "/" || $0 == "\\" })
       .map { rest.index(after: $0) } ?? rest.startIndex
     return rest[afterSlashes...]
   }
 
-  var libNames = Array(
-    linkLibraries
-      .filter { $0.hasPrefix("LLVM") && $0.hasSuffix(".lib") }
-      .map { LinkerSetting.linkedLibrary(String($0.dropLast(4))) })
-  libNames.append(.linkedLibrary("ntdll"))
-  return libNames
+  return Array(linkLibraries.map { LinkerSetting.linkedLibrary(String($0.dropSuffix(".lib"))) })
 }
 
-let llvmLinkerSettings = osIsWindows ? windowsLinkerSettings() : []
+let llvmLinkerSettings =
+  osIsWindows
+  ? windowsLinkerSettings()
+  : [
+    .unsafeFlags(["-L/opt/homebrew/lib"], .when(platforms: [.macOS]))
```

Comment:
I'm concerned about the homebrew thing.  Does that mean we now have a dependency on homebrew?  Dimi uses macports instead.  Why is this needed?  There should be a comment.